pkg/kubelet/status/status_manager.go

```diff
@@ -255,10 +298,7 @@ func (m *manager) SetPodAllocation(pod *v1.Pod) error {
 	m.podStatusesLock.RLock()
 	defer m.podStatusesLock.RUnlock()
 	for _, container := range pod.Spec.Containers {
-		var alloc v1.ResourceList
-		if container.Resources.Requests != nil {
-			alloc = container.Resources.Requests.DeepCopy()
-		}
+		alloc := *container.Resources.DeepCopy()
 		if err := m.state.SetContainerResourceAllocation(string(pod.UID), container.Name, alloc); err != nil {
```

Comment:
Rename? SetContainerResourceAllocation -> SetContainerResources.

Reply:
I'm planning a follow-up PR to refactor the allocation logic out of status manager to something like a new "allocation manager". Mind if I defer this to that PR?